### DISCUSSION TITLE
リスト内のNote記法がうまく閉じられない

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#303] [要望・アイデア・不具合]
リスト内に *Note* 記法をネストすると、ネスト内で *Note* 記法を閉じることが出来ません。これは、順序無しでも 順序付きリストでも結果は同じです。

![Note in list](https://user-images.githubusercontent.com/75878492/190873491-3b8208ef-1368-4078-8833-1b77d38c19c3.png)

### サンプルソース

<details><summary>sample</summary>

```markdown
- 終了符号のない *Note* 記法

    :::note info
    *Note* の終わりを示す `:::` が無くても *Note* が閉じます。

- 終了符号のある *Note* 記法

    :::note warn
    *Note* の終わりを示す `:::` があっても *Note* が閉じません。代わりに謎の `::` が表示されます。
    :::

    終了しないので、ここも *Note* 内です。

    :cry:

- Non-nested list
```

</details>

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
ご報告ありがとうございます！
認識できていなかったので、調査いたします🙇
